packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -25,6 +30,10 @@ import {
 } from "@fluid-internal/test-version-utils";
 import { v4 as uuid } from "uuid";
 import { ConfigTypes, IConfigProviderBase } from "@fluidframework/telemetry-utils";
+// eslint-disable-next-line import/no-internal-modules
+import { PendingLocalState } from "@fluidframework/container-runtime/dist/test";
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
we should avoid needing internals in our e2e tests

Reply:
Now I'm mocking the createBlob storage call but that doesn't change these imports. They are both types which I use while verifying pending blob list, I could use "any" instead.